native/game_backend/src/player.rs

```diff
@@ -15,6 +16,8 @@ use crate::loot::Loot;
 use crate::map;
 use crate::map::Position;
 
+const BASIC_SKILL_KEY: &str = "1";
```

Comment:
This is essentially hardcoding things to a name in the config, the main thing we need to avoid doing in the backend

Reply:
Right, I removed it as part of the responsibilities refactor